nixos/modules/system/boot/systemd.nix

```diff
@@ -749,6 +749,22 @@ in
       '';
     };
 
+    systemd.tmpfiles.packages = mkOption {
+      type = types.listOf types.package;
+      default = [];
+      example = literalExample "[ pkgs.lvm2 ]";
+      apply = map getLib;
+      description = ''
+        List of packages containing <command>systemd-tmpfiles</command> rules.
+
+        All found files in
+        <filename><replaceable>pkg</replaceable>/lib/tmpfiles.d</filename>
+        will be included.
+
+        If a <filename>lib</filename> output is available, rules are searched there.
```

Comment:
This should somehow describe it then /only/ looks in the `lib` output

(and maybe explain the whole fallback depending on which outputs the package has (lib -> out -> default output)), but I'm struggling with how to write this in a more explicit and less misunderstandable way.

Reply:
How about: "If a <filename>lib</filename> output is available, rules are searched there and only there. If there is <filename>lib</filename> it will fall back to <filename>out</filename> and if that does not exist either, the default output will be used."

Although this should maybe just be documented somewhere "generically". As in, explain `getOutput` in some part of the manual and reference if for options like this, because there are probably a bunch of them.

Reply:
Yeah, let's refactor it to a more generic place as soon as there's more than 2 things trying to document it :-)

Reply:
#93006 isn't trying to document it right now, but it also does `apply = map getLib;`

Reply:
Can you change this to "If a lib output is available, rules are searched there and only there. If there is lib it will fall back to out and if that does not exist either, the default output will be used." for now?

Reply:
done